src/content/tutorials/observability/data-ingestion/metrics/_index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Ingest metrics into the Observability Platform
+linkTitle: Metrics
+description: How to ingest new metrics in the Observability Platform
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-metrics
+weight: 50
+last_review_date: 2024-07-11
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - What is a ServiceMonitor?
+  - What is a PodMonitor?
+  - How do I ingest new metrics?
+  - Why do my clusters run the Prometheus Operator?
+---
+
+By default, all Giant Swarm clusters are equiped with the [Prometheus Operator](https://prometheus-operator.dev/) and a set monitoring agents (multiple [Prometheus](https://prometheus.io/) shards in `agent mode`) to be able to collect and forward cluster critical and workload metrics to a [Grafana Mimir](https://grafana.com/oss/mimir/) instance running on the management cluster.
+
+As no workload is the same, especially in the way it exposes its metrics, we decided to base our monitoring configuration on the `ServiceMonitor` and `PodMonitor` CRDs provided by the Prometheus Operator. Those allow us to:
+
+- define where the metrics we want to ingest are (e.g. container, port)
+- transform metrics before ingesting them (e.g. dropping unneeded data, adding extra labels)
+
+If you want to read more about the CRDs, feel free to look at their [API](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/api.md).
+
+## Prerequisites
+
+Before you start to ingest data from a container running in your clusters, you need to make sure that your application is already [instrumented](https://opentelemetry.io/docs/concepts/instrumentation/) to export metrics and make sure the metrics it provides are useful to you.
+
```

Comment:
I like the "make sure the metrics it provides are useful to you."

Maybe here we could add the part with the "more metrics means more resource cost" like

> Keep in mind that ingesting new metrics into the Observability Platform comes with a cost. The resource consumption of the central Mimir is related to the amount of metrics it has to handle. This means ingesting more metrics also leads to higher resource consumption of the Observability Platform overall.
> 
> You can check the resource usage related to your `ServiceMonitor` and `PodMonitor` in the Grafana Dashboard \<Name of the Resource Usage per Service Monitor Dashboard> 

Forgot the name of the dashboard :D But yeah something in that direction sure is helpful here.

Reply:
The dashboards are https://github.com/giantswarm/docs/pull/2247/files#diff-982e0af9b9f3200b73a166ff7f752289fe106c848e963bd411a8a99734c4f821R65 :D 

Do we want to name them again here so going with this suggestion?

```suggestion

Keep in mind that ingesting new metrics into the Observability Platform comes with a cost. The resource consumption of the central Mimir is related to the amount of metrics it has to handle. This means ingesting more metrics also leads to higher resource consumption of the Observability Platform overall.

You can check the resource usage related to your ServiceMonitor and PodMonitor in the  `ServiceMonitors Overview` Dashboard  in your installations Grafana.
```